cmd/zed/zed.h

```diff
@@ -15,11 +15,6 @@
 #ifndef	ZED_H
 #define	ZED_H
 
-/*
- * Absolute path for the default zed configuration file.
- */
-#define	ZED_CONF_FILE		SYSCONFDIR "/zfs/zed.conf"
```

Comment:
This is functionality we'd always intended to add and thus has stubbed out.  Though clearly we haven't gotten to it.  In particular, in `zfs_diagnosis.c` there are two functions `fmd_prop_get_int32` and `fmd_prop_get_int64` which return hard coded values we thought could be set in the config file.  That said, at this point since we're only talking about a couple of value it might make more sense to add a few command line options to override the defaults.  So I'm OK with shedding this long dormant and partial config file support.